doc/source/data/training_ingest.rst

```diff
@@ -0,0 +1,525 @@
+.. _training_ingest_home:
+
+End-to-end: Data Loading for ML Training
+========================================
+
+Data loading & preprocessing is an important step for ML model training. The process involves loading raw data from storage, processing it to the appropriate format, sharding it among training workers, and moving each shard to GPU for training.   
+
+:ref:`Ray Data <data>` works with :ref:`Ray Train <train-docs>` to offer an efficient solution to load and preprocess data for distributed training in a streaming fashion, scaling to very large datasets and avoiding GPU starvation.
+
+For an overview on why you should use Ray Data to ingest data for training, and how it compares to alternatives, see the :ref:`Ray Data Overview <data_overview>`.
+
+.. figure:: images/train_ingest.png
+
+.. _ingest_quickstart:
+
+Quickstart
+----------
+To start, install Ray Data and Ray Train:
+
+.. code-block:: bash
+
+    pip install -U "ray[data,train]"
+
+Using Ray Data and Ray Train for distributed training on large datasets involves 4 basic steps:
+
+- **Step 1:** Load your data into a Ray Dataset. Ray Data supports many different data sources and formats. For more details, see :ref:`Loading Data <loading_data>`.
+- **Step 2:** If required, create a function that defines your training logic and periodically checkpoints your model. For more information, see the :ref:`Ray Train user guides <train-userguides>`.
+- **Step 3:** Inside your training function, access the dataset shard for the training worker via :meth:`session.get_dataset_shard() <ray.air.session.get_dataset_shard>`. Iterate over the dataset shard to train your model. For more details on how to iterate over your data, see the :ref:`Iterating over data <iterating-over-data>` user guide.
+- **Step 4:** Create your :ref:`Ray Train Trainer <train-framework-catalog>` and pass in your Ray Dataset. This will automatically shard the datasets and pass them to each training worker. For more information on configuring training, see the :ref:`Ray Train user guides <train-userguides>`.
+
+* For more in-depth examples for your use case, see :ref:`our distributed training examples<data_train_examples>`.
+* For how to configure Ray Datasets for training, see :ref:`the configuration guide <train_datasets_configuration>`.
+* For more information on how to configure Ray Train in general, see :ref:`the Ray Train user guides <train-userguides>`
```

Comment:
Feel like this is somewhat out of place in the layout 
<img width="784" alt="image" src="https://github.com/ray-project/ray/assets/26107013/ea6833ed-a71e-44f1-942b-09ac0cb505c9">

I'm wondering if it'd make sense to merge this with "More examples" and rename the section "Related resources" or something?